pkg/addons/registry/registry.go

```diff
@@ -0,0 +1,229 @@
+package registry
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+	"github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"golang.org/x/crypto/bcrypt"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/airgap"
+	"github.com/replicatedhq/embedded-cluster/pkg/helpers"
+	"github.com/replicatedhq/embedded-cluster/pkg/kubeutils"
+	"github.com/replicatedhq/embedded-cluster/pkg/spinner"
+)
+
+const (
+	releaseName = "docker-registry"
+	namespace   = "registry"
+)
+
+// Overwritten by -ldflags in Makefile
+var (
+	ChartURL     = "https://url"
+	ChartName    = "name"
+	Version      = "v0.0.0"
+	ImageVersion = "2.8.3"
+)
+
+var registryPassword = helpers.RandString(20)
+var registryAddress = ""
+
+var helmValues = map[string]interface{}{
+	"replicaCount":     1,
+	"fullnameOverride": "registry",
+	"image": map[string]interface{}{
+		"tag": ImageVersion,
+	},
+	"storage": "hostpath", // this is not a recognized option but gets us past all the storage options
```

Comment:
Where does the data end up in this case?

Reply:
This would be a `filesystem` then. I think you're getting that value by default.

Reply:
Might be easier to switch to s3/seaweedfs with your current format later though :)

Reply:
filesystem created a PVC IIRC